new/PRS14427Williams.xml

```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14427Williams" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>C. L. G. Williams</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-05-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName><forename>C.</forename> <forename>L.</forename> <forename>G.</forename> 
+                        <surname>Williams</surname></persName>
+                    <floruit notBefore="1900" notAfter="1990">20th century</floruit>
```

Comment:
see https://issuu.com/jammdesign/docs/dw101-095 p. 92 no. 439
![image](https://github.com/BetaMasaheft/Persons/assets/17987728/84f93a8d-890a-4cb7-8744-887213563e43)

on the skinner's cavalry

`Skinner's Horse in World War-II
In September 1939, when World War II was declared, Skinner's Horse was still a mounted regiment. When the mobilisation regulations were opened, one of the first orders was: Swords will be sharpened! That the regiment was able to take its place as a reconnaissance unit as a part of a motorised brigade in East Africa within nine months and was in action as early as the autumn of 1940 was a matter of pride to all ranks.`
https://premium.globalsecurity.org/military/world/india/skinners-horse.htm

so he was a military captain in British East Africa during WW2

Reply:
Thank you for your investigation!